client/client.go

```diff
@@ -51,6 +54,28 @@ type queryValue struct {
 	name, value string
 }
 
+func Infof(format string, args ...interface{}) {
+	if glog.V(1) {
+		glog.InfoDepth(1, fmt.Sprintf("ciao-cli INFO: "+format, args...))
+	}
+}
+
+func Errorf(format string, args ...interface{}) {
+	glog.ErrorDepth(1, fmt.Sprintf("ciao-cli ERROR: "+format, args...))
+}
+
+func Fatalf(format string, args ...interface{}) {
```

Comment:
I think these changes can actually be dropped, ignore for now since the SDK is not printing anything anymore and only returning data.

Reply:
Yes please don't add these.